packages/gestalt-codemods/16.0.0/upsell-primary-secondary-action-replace-primary-secondary-link.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Converts
+ *  <Upsell primaryLink={...} secondaryLink={...} /> to <Upsell primaryAction={...} secondaryAction={...} />
+ */
+
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const src = j(file.source);
+  let localIdentifierName;
+  let fileHasModifications = false;
+
+  src.find(j.ImportDeclaration).forEach((path) => {
+    const decl = path.node;
+    if (decl.source.value !== 'gestalt') {
+      return null;
+    }
+
+    localIdentifierName = decl.specifiers
+      .filter((node) => node.imported.name === 'Upsell')
```

Comment:
we could merge two codemods in one by doing

`.filter((node) => ['Upsell, 'Callout'].includes(node.imported.name))`